src/components/App.js

```diff
@@ -1,5 +1,17 @@
+/* eslint-disable arrow-body-style */
 import React from 'react';
 
-const App = () => <div>App</div>;
+import BooksList from '../containers/BookList';
+import BookForm from '../containers/BookForm';
+
+const App = () => {
+  return (
+    <div>
+      <BookForm />
+      <br />
+      <BooksList />
+    </div>
+  );
+};
```

Comment:
Instead of this disabling this, we could refactor the component since in this case, we don't need an implicit `return` keyword.

Something like this could solve the linter warning.

```jsx
const App = () => (
  <div>
    <BookForm />
    <br />
    <BooksList />
  </div>
);
```